src/Microsoft.Sbom.Api/Workflows/SbomConsolidationWorkflow.cs

```diff
@@ -92,15 +95,38 @@ private IEnumerable<ConsolidationSource> GetSbomsToConsolidate(string artifactPa
             return null;
         }
 
-        return detectedSboms.Select((sbom) => new ConsolidationSource(info, sbomConfigFactory.Get(sbom.manifestInfo, manifestDirPath, metadataBuilderFactory)));
+        return detectedSboms.Select((sbom) => new ConsolidationSource(info, sbomConfigFactory.Get(sbom.manifestInfo, manifestDirPath, metadataBuilderFactory), artifactPath));
     }
 
     private async Task<bool> ValidateSourceSbomsAsync(IEnumerable<ConsolidationSource> consolidationSources)
     {
-        var validationWorkflows = consolidationSources
-            .Select(async sbom => await Task.FromResult(true)); // TODO: Run validation workflow
-        var results = await Task.WhenAll(validationWorkflows);
-        return results.All(b => b);
+        var result = true;
+        foreach (var source in consolidationSources)
+        {
+            var identifier = Math.Abs(string.GetHashCode(source.SbomConfig.ManifestJsonFilePath)).ToString();
+            var defaultOutputPath = configuration.OutputPath.Value;
+            var workflowResult = false;
+            try
+            {
+                configuration.ValidateSignature.Value = !source.ArtifactInfo.SkipSigningCheck ?? true;
+                configuration.IgnoreMissing.Value = source.ArtifactInfo.IgnoreMissingFiles ?? false;
+                configuration.BuildDropPath.Value = source.BuildDropPath;
+                configuration.OutputPath.Value = fileSystemUtils.GetTempFile($"validation-results-{identifier}.json");
+
+                Console.WriteLine($"Running validation for {source.SbomConfig.ManifestJsonFilePath} with identifier {identifier}. Writing output results to {configuration.OutputPath}.");
```

Comment:
Will this eventually go through the logger?

Reply:
I'm unsure- the validation workflow does [pretty much all of its printing via the console, not the logger](https://github.com/microsoft/sbom-tool/blob/main/src/Microsoft.Sbom.Api/Workflows/SbomParserBasedValidationWorkflow.cs#L234), which is obviously not ideal. That said, I'm following that pattern here because IMO by default this line should have the same verbosity as all of the validation results that get printed to the console, and I don't think we can achieve that with the logger since by default only warnings and errors written to the logger are displayed. Feels like we could do a separate feature to clean up all of the logging vs console stuff throughout this repo, but I don't want to do a overhaul of that right now.

Reply:
It looks like this pattern was set in #274, although no reason or explanation is given in the PR. Our default log level is `Information`, so I would think that the most correct answer is to use `logger.Information`. Other workflows seem to reserve `Console.WriteLine` to specific error cases.

All that said, this feels like a separate conversation/PR about how we want to display the output holistically